.ci/scripts/util/secrets_scan_compare.sh

```diff
@@ -23,9 +23,19 @@ then
     echo "" >&2
     echo "Please follow the steps below on your local machine to reveal and handle the secrets:" >&2
     echo "" >&2
-    echo "1️⃣ Run the 'detect-secrets' tool on your local machine. This tool will identify and clean up the secrets. You can find detailed instructions at this link: https://nasa-ammos.github.io/slim/continuous-testing/starter-kits/#detect-secrets" >&2
+    echo "1. Run the 'detect-secrets' tool on your local machine. This tool will identify and clean up the secrets:" >&2
     echo "" >&2
-    echo "2️⃣ After cleaning up the secrets, commit your changes and re-push your update to the repository." >&2
+    echo "    \$ detect-secrets scan --all-files --disable-plugin AbsolutePathDetectorExperimental --exclude-files '\.secrets\..*' --exclude-files '\.git.*' --exclude-files 'test_results' --exclude-files '\.pytest_cache' --exclude-files '\.venv' --exclude-files 'venv' --exclude-files 'dist' --exclude-files 'build' --exclude-files '.*\.egg-info' > .secrets.baseline" >&2
+    echo "" >&2
+    echo "2. Perform an audit on the updated .secrets.baseline to disposition any new findings:" >&2
+    echo "" >&2
+    echo "    \$ detect-secrets audit .secrets.baseline" >&2
+    echo "" >&2
+    echo "The audit tool will ask the following for all newly detected \"secrets\": \"Is this a secret that should be committed to this repository?\"" >&2
+    echo "" >&2
+    echo "If the detected secret is benign (i.e. a public email address or dummy password) the correct answer is \"y\"." >&2
```

Comment:
Maybe there should be a little more attention called out to this line considering how counterintuitive it is?

Reply:
I'd imagine you can just put the emoji characters directly into the .sh file 
I just tried this in PyCharm and it worked no problem

Barring that, you could try a plain text approach like even a `NOTE: ` before the message.

Perhaps move the note to before the step with the audit command so users are aware that `'y' == safe` _before_ running the command